cpp/ycm/tests/CMakeLists.txt

```diff
@@ -33,6 +33,11 @@ if ( USE_SYSTEM_GMOCK )
   find_package( GTest REQUIRED )
   find_package( GMock REQUIRED )
 else()
+  if ( WIN32 )
+    # Override BUILD_SHARED_LIBS option in gmock and gtest CMakeLists
+    set( BUILD_SHARED_LIBS ON CACHE BOOL "Build shared libraries (DLLs)." )
```

Comment:
Why? I've used cmake before with gtest/gmock without resorting to shared libs. [See the FlightCrew source code](https://code.google.com/p/flightcrew/source/browse/#git) (which I wrote years ago).